Templates/DefaultProject/Template/ExportScripts/export_standalone_monolithic.py

```diff
@@ -0,0 +1,321 @@
+    bundles_path = project_path / 'AssetBundling' / 'Bundles'
+
+    #generate the asset lists for the project
+    gen_game_asset_list_command = \
+        f"{asset_bundler_batch_path} assetLists --assetListFile {game_asset_list_path} --platform {selected_platform} --allowOverwrites"
+    for seed in seedlist_paths:
+        gen_game_asset_list_command += " --seedListFile " + str(seed)
+    process_command(gen_game_asset_list_command.split())
+
+    gen_engine_asset_list_command = \
+        f"{asset_bundler_batch_path} assetLists --assetListFile {engine_asset_list_path} --platform {selected_platform} --allowOverwrites --addDefaultSeedListFiles"
+    process_command(gen_engine_asset_list_command)
+
+    #generate the bundles. We will place it in the project directory for now, since the files need to be copied multiple times (one for each separate launcher distribution)
+    process_command([asset_bundler_batch_path, "bundles", "--maxSize", "2048", "--platform", selected_platform, "--allowOverwrites",
+                         "--outputBundlePath", bundles_path / f"game_{selected_platform}.pak",
+                         "--assetListFile", game_asset_list_path])
+    
+    process_command([asset_bundler_batch_path, "bundles", "--maxSize", "2048", "--platform", selected_platform, "--allowOverwrites",
+                         "--outputBundlePath", bundles_path / f"engine_{selected_platform}.pak",
+                         "--assetListFile", engine_asset_list_path])
+    return bundles_path
+
+
+def create_launcher_layout_directory(project_path: pathlib.Path,
+                                     output_path: pathlib.Path,
+                                     platform: str,
+                                     mono_build_path: pathlib.Path,
+                                     bundles_to_copy: List[pathlib.Path],
+                                     gem_folder_names_to_copy: List[str],
+                                     project_file_patterns_to_copy: List[str],
+                                     build_config: str,
+                                     archive_output_format: str = "none",
+                                     logger: logging.Logger|None =None):
+    
+    output_cache_path = output_path / 'Cache' / platform
+    
+    if output_path.exists():
+        shutil.rmtree(output_path)
+
+    os.makedirs(output_cache_path, exist_ok=True)
+
+    for bundle in bundles_to_copy:
+        shutil.copy(bundle, output_cache_path)
+
+    for file in glob.glob(str(pathlib.PurePath(mono_build_path / 'bin' / build_config / '*.*'))):
+        shutil.copy(file, output_path)
+
+    for gem_folder_name in gem_folder_names_to_copy:
+        output_gem_path = output_path / 'Gems'/ gem_folder_name
+        os.makedirs(output_gem_path, exist_ok=True)
+        for file in glob.glob(str(pathlib.PurePath(mono_build_path / 'bin' / build_config / 'Gems' / gem_folder_name / '*.*'))):
```

Comment:
Gem Folders names don't have to be unique and are not in the project gem case(notice the folder is called "Gem" in most cases).

What is unique for a Gem is it's name.
So the gem name should be used as the output folder to prevent two gems with the same folder name from colliding.

The Gem name can be query via getting the gem.json file by using by invoking [manifest.get_gem_json_data](https://github.com/o3de/o3de/blob/development/scripts/o3de/o3de/manifest.py#L948-L949) method.
```python
from o3de import manifest
# ...
gem_json_data = manifest.get_gem_json_data(gem_path=path_to_gem)
gem_name = gem_json_data.get('gem_name', gem_folder_name) if gem_json_data else gem_folder_name
```